web-registry/src/pages/Projects/utils/sortProjects.ts

```diff
@@ -1,28 +1,5 @@
-import { useEffect, useState } from 'react';
-
 import { ProjectWithOrderData } from '../Projects.types';
 
-type Props = {
-  projects: ProjectWithOrderData[];
-  sort: string;
-};
-
-export const useSortProjects = ({
-  projects,
-  sort,
-}: Props): ProjectWithOrderData[] => {
-  const [sortedProjects, setSortedProjects] =
-    useState<ProjectWithOrderData[]>(projects);
-
-  useEffect(() => {
-    const projectsCopy = [...projects]; // to make sure render picks up change
-    const _sortedProjects = sortProjects(projectsCopy, sort);
-    setSortedProjects(_sortedProjects);
-  }, [projects, sort]);
-
-  return sortedProjects;
-};
```

Comment:
This was the effect that was causing rerender. We're now using sort functions directly instead.